src/System.Private.CoreLib/shared/System/ComponentModel/DefaultValueAttribute.cs

```diff
@@ -23,6 +20,21 @@ public class DefaultValueAttribute : Attribute
         /// </devdoc>
         private object _value;
 
+        // We cache "reflection" types for conversion fallback
+        static Type s_typeDescriptorTypeCached;
+        static MethodInfo s_convertFromInvariantStringMethodCached;
+
+        static DefaultValueAttribute()
+        {
+            // We discover/cache types for conversion fallback on first load
+            s_typeDescriptorTypeCached = Type.GetType("System.ComponentModel.TypeDescriptor, System, Version=0.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089", throwOnError: false);
```

Comment:
This should not be in static constructor. This should be done as lazy as possible only when it is needed.

Reply:
could i use Lazy<T> or usually there are other pattern con codebase?

Reply:
Here is the pattern to follow: https://github.com/dotnet/corefx/blob/b9d82ae7fd6912b6c0c758e47d0824689adaeafc/src/System.Runtime.Extensions/src/System/AppDomain.cs#L409